packages/aila/src/core/document/types.ts

```diff
@@ -0,0 +1,11 @@
+import type { LooseLessonPlan } from "../../protocol/schema";
+
+export type AilaDummyDocumentContent = {
```

Comment:
This is an example of a different type of document that Aila could produce.

I think we will probably want a discriminating "type" attribute so we can use Zod to do a discriminated union. Not implemented yet because that would change the shape of the LooseLessonPlan.